src/bartiq/symbolics/ast_parser.py

```diff
@@ -271,8 +276,12 @@ def _(value_node: ast.Call):
     assert isinstance(value_node.func, ast.Name)
     if value_node.func.id != "wildcard":
         raise ValueError("Should never encounter function call other than wildcard() in the attribute lookup")
-    assert isinstance(value_node.args[0], ast.Name)
-    return f"{value_node.args[0].id}~" if value_node.args else "~"
+    # We can't use ternary operator here because it confuses mypy
```

Comment:
I'd remove this comment – If I just read it, it would confuse me a bit.
And if someone ever tries to refactor it to be more concise, they'll quickly see why it's done the way it is.
WDYT?

Reply:
I left this comment because I personally would be confused if I've seen if-else statement of this form instead of a ternary operator - but I agree it might be unnecessary. I removed it.